tensorflow/python/summary/summary.py

```diff
@@ -129,6 +129,10 @@ def image(name, tensor, max_outputs=3, collections=None, family=None):
     tensor: A 4-D `uint8` or `float32` `Tensor` of shape `[batch_size, height,
       width, channels]` where `channels` is 1, 3, or 4.
     max_outputs: Max number of batch elements to generate images for.
+    vmin: Maximum clipping value.
```

Comment:
Recommend a more verbose comment, along the lines of how matplotlib describes vmin and vmax in https://matplotlib.org/devdocs/api/_as_gen/matplotlib.pyplot.imshow.html
Also describe what happens if the user specifies only one of these two parameters.

Reply:
yeah, add more description at the comments.